src/tests/ftest/control/dmg_system_cleanup.py

```diff
@@ -85,6 +86,8 @@ def test_dmg_system_cleanup_one_host(self):
         expected_count = {pool.uuid.lower(): 6 for pool in self.pool}
 
         # Clear pool and container list to avoid trying to destroy them.
+        for pool in self.pool:
+            pool.skip_cleanup()
```

Comment:
what is the reason for skipping cleanup on the pools?

Reply:
I would let @phender comment on this.